files/domain-validation.sh

```diff
@@ -4,23 +4,23 @@
 # First parameter is the certificate path
 # All other parameters are the domain names requested for the certificate
 # Split whitespace into newlines to let it play nicely with sort
-certpath="${1}"
+certpath="$1"
```

Comment:
why do you remove all the braces? They are fine and don't do any harm.

Reply:
Shellcheck wants the variables in quotes. "$var" is a lot easier to type than "${var}". I changed the rest of them to be consistent with the vars I added. I can reverse it if you like.

Reply:
I agree with the quotes, but the braces aren't related. Can you please readd them? That makes the diff easier to read.

Reply:
Sure. :) Done. Vars are now consistent the other way, -with- braces.